src/lib/AnnotationControls.ts

```diff
@@ -194,7 +195,13 @@ export default class AnnotationControls {
     /**
      * Initialize the annotation controls with options.
      */
-    public init({ fileId, onEscape = noop, onClick = noop, showHighlightText = false }: Options): void {
+    public init({
+        enableAnnotationsImageDiscoverability = false,
```

Comment:
It doesn't seem like this is what we should do, we try to keep `AnnotationControls` as a dumb component that just renders the buttons based on state. If we want to initialize the controls in a certain state, then we should just provide a `initialMode` or something more generic

Reply:
I see. So maybe we can instead pass in `initialMode` in as a new parameter in the init method?